mix.exs

```diff
@@ -1,27 +1,14 @@
-defmodule Mix.Tasks.Compile.Exfswatch do
```

Comment:
because the logic becomes more complex than before, if I put compiling method of all backends in a single function, there will be a huge function, so I decide to split it to each backend module.